packages/@glimmer/wire-format/index.ts

```diff
@@ -226,7 +226,9 @@ export function isAttribute(val: Statement): val is Statements.Attribute {
   return (
```

Comment:
On 0.38 the wire format definitions are in this file

Reply:
I don't follow what you mean. I've just cherry picked the commit I merged into master.

Reply:
Nevermind, I just noticed the `d.ts` file on this PR.